service/src/test/java/cz/muni/fi/pa165/mushrooms/service/MushroomHunterServiceTest.java

```diff
@@ -0,0 +1,374 @@
+package cz.muni.fi.pa165.mushrooms.service;
+
+import cz.muni.fi.pa165.mushrooms.dao.MushroomHunterDao;
+import cz.muni.fi.pa165.mushrooms.entity.MushroomHunter;
+import cz.muni.fi.pa165.mushrooms.validation.PersistenceSampleApplicationContext;
+import mockit.Delegate;
+import mockit.Expectations;
+import mockit.Injectable;
+import mockit.Tested;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.AbstractTransactionalJUnit4SpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by Matúš on 26.11.2017.
+ */
+@ContextConfiguration(classes = PersistenceSampleApplicationContext.class)
```

Comment:
this hsould be the new class (config for service, not persistence)

change for @ContextConfiguration(classes = ServiceConfiguration.class)

Reply:
resolved